code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/integration/ldap/OpenLDAP/OpenLDAPIdentityProvider.java

```diff
@@ -123,7 +123,7 @@
                             '=' + memberOf.get(i).toString() + ")(" + getConstantsConfig().objectClass() + '=' +
                             getConstantsConfig().getGroupObjectClass() + "))";
                     logger.trace("getIdentities: memberOf attribute query: "+query);
-                    identities.addAll(resultsToIdentities(searchLdap(query)));
+                    identities.addAll(resultsToIdentities(searchLdap(query, "user")));
```

Comment:
This needs to call searchLdap(query, "group"), since the search query is searching for  groups by looking for  getConstantsConfig().getGroupObjectClass()